packages/ui-components/src/components/DataTable/DataTable.tsx

```diff
@@ -28,7 +37,19 @@ const getColumnId = ({ id, accessor, Header }) => {
   return Header;
 };
 
-export const DataTable = ({ columns, data, className, rowLimit, total }) => {
+export const DataTable = ({
+  columns,
+  data,
+  className = '',
+  rowLimit = 0,
+  total = 0,
+}: {
```

Comment:
Would you mind making this into a DataTableProps interface as I think that's easier to read when there are more than 2 or 3 props